src/ResourceManager/Network/ChangeLog.md

```diff
@@ -19,10 +19,9 @@
 -->
 ## Current Release
 
-## Version 3.1.0
```

Comment:
@Nilambari Hey Nilambari, please keep the information corresponding to release 3.1.0 in the change log so that customers looking at the change log can follow changes across multiple releases. Also, please move the information under "Version 3.2.0" to under "Current Release". At the end of the release, we run a script that will move the contents from under "Current Release" and put them under a newly created release section (in this case, 3.2.0).